docs/processing_information.md

```diff
@@ -0,0 +1,64 @@
+# Processing Information
+
+## Single Cell and Single Nuclei Samples
```

Comment:
This header seemed ambiguous for the info included. I also added hyphens because that seemed to be standard elsewhere, and changed to sentence case for the header, which I prefer, but we should make a decision about.
```suggestion
## Single-cell and single-nuclei RNA-seq
```